### PR TITLE
Add aicsimageio

### PR DESCRIPTION
**Pull request by tlambert03**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JacksonMaxfield** [comment]
Can confirm 👍

**Comment by tlambert03** [comment]
@toloudis ... please also confirm when you get a chance that you're willing to be listed in meta.yaml as a maintainer.  @JacksonMaxfield ... any other names I should put on there?

File: recipes/aicsimageio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pytest-runner >=5.2
+  run:
+    - python >=3.7
```
Comment (goanpeca) [inline recipes/aicsimageio/meta.yaml:23]: It seems all these are needed @tlambert03 ?

https://github.com/AllenCellModeling/aicsimageio/blob/main/setup.py#L81

Comment (tlambert03) [reply recipes/aicsimageio/meta.yaml:23]: yes, but only because of pytest-runner... I'm chatting with @JacksonMaxfield about this at the moment :)

Comment (tlambert03) [reply recipes/aicsimageio/meta.yaml:23]: oh sorry... nevermind, I'm debugging something else!

**Comment by toloudis** [comment]
This looks like it's fixed at aicsimageio 4.1.  Do we automate new releases on the `aicsimageio` repo side?  I don't know how this all works :)

**Comment by tlambert03** [comment]
> This looks like it's fixed at aicsimageio 4.1. Do we automate new releases on the aicsimageio repo side? I don't know how this all works :)

once this PR is accepted/merged by conda-forge, a new "feedstock" repository will be created, and you will be added as a member. (see the [napari feedstock](https://github.com/conda-forge/napari-feedstock) as an example).  Every time you push a new release to pypi, the amazing conda forge bots will open a PR to that feedstock repository updating the recipe (`meta.yaml`) with the new version available on PyPI, along with the sha checksum to the file available for download on pypi.  You have a chance at that point to update the recipe (confirm stuff and dependencies and what not), once the PR passes and you're happy with it, you merge it and then the bots do their amazing work to build the conda package, upload it to anaconda cloud, and make the new version available at `conda install -c conda-forge aicsimageio`

**Comment by tlambert03** [comment]
this is ready for review @goanpeca (fun to be able to bother you directly here now 😂 !) @conda-forge/help-python

**Comment by toloudis** [comment]
Confirmed, I accept being listed as a maintainer.

**Comment by goanpeca** [comment]
All looks good. Thanks for the recipe @tlambert03 :)

**Comment by tlambert03** [comment]
thank you!

**Pull request merged**